charts/argo-workflows/templates/server/server-service.yaml

```
@@ -17,6 +17,9 @@ spec:
     name: {{ . }}
     {{- end }}
     targetPort: 2746
+    {{- if eq .Values.server.serviceType "NodePort" }}
```

Let's use the same conditional as we use for LoadBalancer some lines below:
```suggestion
    {{- if and (eq .Values.server.serviceType "NodePort") .Values.server.serviceNodePort }}
```

---

Resolved, as per PR update